shaders/anim_basic/frag.glsl

```diff
@@ -56,15 +57,15 @@ void main()
 vec3 calc_color(Material mat, vec3 normal, vec3 view_dir, vec3 light_dir, vec3 light_intensity, float ambient_coeff)
```

Comment:
There is indeed code duplication in our shader code currently. More specifically ``anim_basic/frag`` is equivalent to ``basic/frag``. And ``anim_unlit/frag`` == ``unlit/frag``. And ``anim_basic/vert`` == ``anim_unlit/vert``. Same with ``unlit/vert`` and ``basic/vert``.

Reason being: objects that use animation shaders need to change their vertices in the shader, so they do it in their "vert"ex.glsl shader component. But their "frag"ment shaders remain the same because they are lighted/colored in the same way.
Likewise, unlit objects (no shadows or lighting calculations) have different fragment shaders from lit objects, but the same vertex shader.